packages/web/src/components/tile/Tile.module.css

```diff
@@ -1,12 +1,18 @@
 .root {
+  --border-width: 1px;
   display: flex;
   flex-direction: column;
   position: relative;
-  border: 1px solid var(--neutral-light-8);
+  border: var(--border-width) solid var(--neutral-light-8);
   border-radius: var(--unit-2);
   background-color: var(--white);
   transition: all var(--expressive);
-  overflow: hidden;
+}
+
+.borderOffset {
+  position: absolute;
+  top: calc(-1 * var(--border-width));
```

Comment:
Open to feedback on this pattern. My goal with this PR was to remove knowledge of the parent from DogEar. The place where we decide how far to offset a component to cover the border is the place that defines said border!

Reply:
If you set a class or inline style on any component using this class, and within that class/style set `--border-width: value`, this will pick up that value and use it. This is also inheriting the value from the nearest parent that has it set (in this case `.root`), so you can set the value on that instead.

If you were trying to do it from outside this module or the corresponding component, it might be a little trickier. But I think the idea is that this stays contained to only this module.